tests/playground.html

```diff
@@ -320,6 +320,11 @@
 #importExport {
   font-family: monospace;
 }
+
+.ioLabel .blocklyFlyoutLabelText {
```

Comment:
If this is a direct child (and I think it is) then this should be .ioLabel>.blocklyFlyoutLabelText which is much faster to parse.